check-diff.sh

```diff
@@ -548,7 +547,7 @@ function run_phpunit_local {
 			if [ -n "$PHPUNIT_CONFIG" ] || [ -e phpunit.xml* ]; then
 				phpunit $( if [ -n "$PHPUNIT_CONFIG" ]; then echo -c "$PHPUNIT_CONFIG"; fi )
 			fi
-			for nested_project in $( find $PATH_INCLUDES -name 'phpunit.xml*' -mindepth 2 ! -path '*/vendor/*' -name 'phpunit.xml*' -exec dirname {} \; ); do
+			for nested_project in $( find $PATH_INCLUDES -name 'phpunit.xml*' ! -path '*/vendor/*' -name 'phpunit.xml*' -exec dirname {} \; ); do
```

Comment:
That's a great point! 

So we need an alternative way to skip running PHPUnit a second time, specifically in the context of a plugin project and not a site project. One way to do it would be to check in the `for` loop if `$PHPUNIT_CONFIG` is the same as `$nested_project/phpunit.xml*`, and if so, `continue` to skip that iteration.